catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -67,11 +67,22 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
             this.registry = options.registry;
             this.modelBinder = new Backbone.ModelBinder();
             this.mode = options.mode;
+            this.type = options.registryType;
         },
         onRender: function() {
 
-            //TO DO - will want to get any registry details/properties here
-            var config = this.model.get('registryConfiguration').at(0);
+            var config;
+            var type = this.type;
+            if(this.mode === 'add'){
+                this.model.get('registryConfiguration').forEach(function(regConfig){
```

Comment:
Looks like you repeat this same loop in several places. consider adding a function to the model to do this for you and just call it instead. `config = this.model.getConfigForType(type)`